backend/ftp/ftp.go

```diff
@@ -174,6 +175,18 @@ Enabled by default. Use 0 to disable.`,
 If this is set and no password is supplied then rclone will ask for a password
 `,
 			Advanced: true,
+		}, {
+			Name:    "socks_proxy",
+			Default: "",
+			Help: `Socks 5 proxy host for both SSH and SFTP connections.
```

Comment:
This should just say `Socks 5 proxy host.` - it is for the FTP backend so the ssh part isn't relevant.

Reply:
Updated